### ISSUE TITLE
Barostat frequency should be optional

### ISSUE THREAD

**Issue by IAlibay** [OpenFreeEnergy/openfe#712]
Given that gas phase calculations don't use barostats, then it might make sense to make the barostat frequency optional?

**Comment by IAlibay** [comment]
I'm reopening this issue because the underlying problem hasn't been addressed.

Langevin integrators are going to be needed for simulation types where barostats don't make a lot of sense - e.g. implicit solvent. The current approach doesn't really make a lot of sense for users in those scenarios, at least as far as I understand things.

May I ask that the decision makers please explain how they envision things working out for non barostat requiring calculations?

**Comment by richardjgowers** [comment]
@IAlibay yeah we looked at this, it's messy to make it Optional so we just ignored this for now and chose to properly document that Barostats are ignored for our vacuum legs, see: https://github.com/OpenFreeEnergy/openfe/pull/729#pullrequestreview-1884643159

If/when we look at implicit solvent we can look at this again but it didn't seem like a blocker for the release

**Comment by IAlibay** [comment]
Just to say - implicit solvent is going to happen near term, as it's already compatible with the AFE protocol. I have experimental validation work that will need this pretty soon (order month).

**Comment by camattelaer** [comment]
hi

I was looking into the OpenFE documentation but could not find any reference whether or not the ABFE protocol is compatible with implicit solvation. I found this issue which apparently addresses the fact that (in beginning 2024) implicit solvation would be required soon, so i just wanted to ask if the current ABFE implementation in OpenFE would be compatible with an openmm context created with implicit solvation?

Kind regards and thanks in advance

Charles-Alexandre

**Comment by IAlibay** [comment]
Hi @camattelaer - at this moment in time, ABFEs do not support implicit solvation. There is a bit of work in this direction, but it's not currently on our priority list. If this is something of interest to you, could you maybe open an issue discussing your use cases?

**Comment by camattelaer** [comment]
Hi @IAlibay 

Thank you for the very quick reply and clarification!

For some context: I came across the following [preprint](https://arxiv.org/abs/2509.21808), they also have a matching [github](https://github.com/LuchkoLab/Impicit-Solvent-DDM) repo. I am aware that their approach is slightly different from typical ABFE thermodynamic cycles, but i wanted to investigate if there are OpenMM compatible libraries for implicit solvent ABFE, before trying to modify this DDM approach (which is only working with AMBER) to OpenMM.

I will see if opening a new issue is worth both of our trouble. We have not yet definitively decided if we are taking implicit solvent ABFE into consideration.

Kind regards and thanks again for the quick reply!

Charles-Alexandre

**Comment by IAlibay** [comment]
> but i wanted to investigate if there are OpenMM compatible libraries for implicit solvent ABFE, before trying to modify this DDM approach (which is only working with AMBER) to OpenMM.

@camattelaer - probably worth discussing elsewhere, but our current backend for ABFEs *should* allow for implicit solvent (indeed this paper by folks at OpenFF used a very similar bit of underlying code: https://pubs.acs.org/doi/10.1021/acs.jctc.3c01050). The main thing is that it's not currently exposed in OpenFE (aside from some issues with force field parameterisation).
